README.md

```diff
@@ -15,6 +15,61 @@ Please see the [HTSJDK Documentation](http://samtools.github.io/htsjdk) for more
 
 > **NOTE: _HTSJDK does not currently support the latest Variant Call Format Specification (VCFv4.3 and BCFv2.2)._**
 
+#### Building HTSJDK
+
+HTSJDK is now built using [gradle](http://gradle.org/).
+
+A wrapper script (`gradlew`) is included which will download the appropriate version of gradle on the first invocation.
+
+Example gradle usage:
+
+ - compile htsjdk or it's tests  
+ ```
+ ./gradlew compileJava
+ ./gradlew compileTest
+ ```
+ 
+ - build a jar
+ ```
+ ./gradlew jar
```

Comment:
Can we add a default target like we had in ant? Typing "ant" would do a standard build of the library -- could typing "./gradlew" (no args) do a standard build as well (eg., invoke "jar")?